src/alert.js

```diff
@@ -12,40 +12,51 @@ except according to the terms contained in the LICENSE file.
 import { shallowReactive } from 'vue';
 
 class AlertData {
+  #data;
+
   constructor() {
-    this._data = shallowReactive({
+    this.#data = shallowReactive({
       // The alert's "contextual" type: 'success', 'info', 'warning', or
       // 'danger'.
       type: 'danger',
-      message: null,
-      // `true` if the alert should be visible and `false` if not.
-      state: false,
-      // The time at which the alert was last set
+      // Either a string or an array with a component and props for the
```

Comment:
What do you think about having object instead of array? or add `component` and `props` getter so that alert.vue can access them without bracket notation.